tests/unit/test_metrics.py

```diff
@@ -15,10 +15,10 @@
 # Higher numbers = more ways to slice data, more storage, more processing time for summaries
 
 # Cardinality of ctms_requests_total counter
-METHOD_PATH_CODE_COMBINATIONS = 45
+METHOD_PATH_CODE_COMBINATIONS = 47
 
 # Cardinality of ctms_requests_duration_seconds histogram
-METHOD_PATH_CODEFAM_COMBOS = 31
+METHOD_PATH_CODEFAM_COMBOS = 33
 DURATION_BUCKETS = 8
```

Comment:
With 2 new HEAD methods, is this correctly adjusted...?

Reply:
That appears correct to me, adds two new method/path combos